spec/lib/triannon-client/triannon-client_spec.rb

```diff
@@ -23,24 +24,105 @@ def create_annotation
     }
   end
 
+  def delete_annotation(id)
+    tc = TriannonClient::TriannonClient.new
+    tc.delete_annotation(id)
+  end
+
   def check_graph_has_statements(graph)
     expect(graph).to be_instance_of RDF::Graph
-    expect(graph.empty?).to be_falsy
-    expect(graph.size > 2).to be_truthy
+    expect(graph).not_to be_empty
+    expect(graph.size).to be > 2
   end
 
   before :all do
     Dotenv.load
-    @tc = TriannonClient::TriannonClient.new
     @oa_jsonld = '{"@context":"http://iiif.io/api/presentation/2/context.json","@graph":[{"@id":"_:g70349699654640","@type":["dctypes:Text","cnt:ContentAsText"],"chars":"I love this!","format":"text/plain","language":"en"},{"@type":"oa:Annotation","motivation":"oa:commenting","on":"http://purl.stanford.edu/kq131cs7229","resource":"_:g70349699654640"}]}'
   end
 
-  before :each do
-    @anno = create_annotation
+  # before :each do
+  #   @anno = create_annotation
+  # end
+
+  # after :each do
+  #   tc = TriannonClient::TriannonClient.new
+  #   tc.delete_annotation(@anno[:id])
+  # end
+
+  describe 'has constant' do
+    it 'CONTENT_TYPES' do
```

Comment:
if it's a constant and it is an Array of 1 value, would it make more sense for it to be a String DEFAULT_CONTENT_TYPE ?   

Again, not worth a lot of time to change it.